docs/client_reference.rst

```diff
@@ -954,6 +954,11 @@ Response object
 
       HTTP headers of response, :class:`CIMultiDictProxy`.
 
+   .. attribute:: history
+
+      List of :class:`ClientResponse` objects of preceding requests, if there
```

Comment:
Instead of "List" in "List of :class:`ClientResponse` objects …"?

Or where exactly do you mean?